adapters/mediasquare/mediasquare.go

```diff
@@ -0,0 +1,121 @@
+package mediasquare
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/prebid/openrtb/v20/openrtb2"
+	"github.com/prebid/prebid-server/v3/adapters"
+	"github.com/prebid/prebid-server/v3/config"
+	"github.com/prebid/prebid-server/v3/errortypes"
+	"github.com/prebid/prebid-server/v3/openrtb_ext"
+	"github.com/prebid/prebid-server/v3/util/jsonutil"
+)
+
+type adapter struct {
+	endpoint string
+}
+
+func Builder(bidderName openrtb_ext.BidderName, config config.Adapter, server config.Server) (adapters.Bidder, error) {
+	return &adapter{
+		endpoint: config.Endpoint,
+	}, nil
+}
+
+func (a *adapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
+	var (
+		requestData []*adapters.RequestData
+		errs        []error
+	)
+	if request == nil {
+		errs = append(errs, errorWritter("<MakeRequests> request", nil, true))
+		return nil, errs
+	}
+
+	msqParams := initMsqParams(request)
+	msqParams.Test = (request.Test == int8(1))
```

Comment:
I haven't seen this behavior before. Is this added because the msqParams.Test field is required by the MediaSquare backend to recognize the request as a test?

Reply:
Yes, it is.